### PR TITLE
Bump Graffino/Browser-Update v2.0.2 to v2.0.3

### PR DESCRIPTION
**Pull request by phil-davis**
## Related Issue
#36976 

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: build/yarn.lock [hunk 1]
```diff
@@ -29,9 +29,9 @@
   version "1.9.4"
   resolved "https://codeload.github.com/ded/bowser/tar.gz/1fb99ced0e8834fd9662604bad7e0f0c3eba2786"
 
-"@bower_components/browser-update@Graffino/Browser-Update#v2.0.2":
-  version "2.0.2"
-  resolved "https://codeload.github.com/Graffino/Browser-Update/tar.gz/0edda8c7f53ad2cbe68538c00b8c790513096d60"
+"@bower_components/browser-update@Graffino/Browser-Update#v2.0.3":
+  version "3.3.9"
```
Comment (phil-davis) [inline build/yarn.lock]: This is strange. I think that something happened with the `2.0.3` tag in that re[po. It seems to point to a "new" place in that repo that has new code for "future" version 3.3.*

**Comment by phil-davis** [comment]
Also see https://github.com/Graffino/browser-update/pull/1

There is something strange going on in that repo which probably needs to be sorted out first.

**Comment by zenopopovici** [comment]
Repository has been restored and version v2.0.5 has been released to bump the reverted code. Really sorry for the trouble caused.

**Comment by phil-davis** [comment]
> Repository has been restored and version v2.0.5 has been released to bump the reverted code. Really sorry for the trouble caused.

No problem - glad it could be fixed the same day!

**Pull request closed without merging**